dkg/disk.go

```diff
@@ -63,6 +65,8 @@ func loadDefinition(ctx context.Context, conf Config) (cluster.Definition, error
 		if err = json.Unmarshal(buf, &def); err != nil {
 			return cluster.Definition{}, errors.Wrap(err, "unmarshal definition")
 		}
+
+		log.Info(ctx, "Definition file loaded from disk", z.Str("location", conf.DefFile))
```

Comment:
same: `Cluster definition loaded from disk` and move definition hash here

Reply:
done